TehanovAnton/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+ERROR_MESSAGE = "String can't be blank"
+ENTER_PROMPT = 'Enter string:'
+EXIT = 'exit!'
```

Comment:
@TehanovAnton can you please rationalize usage of constants instead of just plain strings?

Reply:
@dmokreckiy What do you mean? Do you mean we should not use constants in this case?